templates/taxi/car_detail.html

```diff
@@ -2,11 +2,23 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Car model: {{ car.model }}
+    <a href="{% url 'taxi:car-update' pk=car.id %}">
+      <i class="bi bi-pencil"></i>
+    </a>
+    <a href="{% url "taxi:car-delete" pk=car.id %}">
```

Comment:
Inconsistent use of quotes in the `{% url %}` tag. It's better to use single quotes consistently, as in `{% url 'taxi:car-delete' pk=car.id %}`.